.sass-lint.yml

```diff
@@ -0,0 +1,18 @@
+rules:
```

Comment:
Linting was dropped in order to reduce the number of dependencies, since it wasn't widely used. Would you mind removing the `.sass-lint.yml` file?